Tutorial: Developing A Flask and ReactJS Application in Okteto

huguestennier opened the pull request:
Port of https://deploy-preview-1230--okteto-website.netlify.app/blog/develop-and-deploy-a-flask-and-reactjs-app-in-okteto/

huguestennier committed:
Add tutorial

huguestennier committed:
remove duplicate image

bgonp commented on src/tutorials/develop-and-deploy-a-flask-and-reactjs-app-in-okteto.mdx:
```diff
@@ -0,0 +1,834 @@
+
+## Scaffolding the frontend application
+
+For the frontend part of this application, you'll be using [Create-React-App](https://reactjs.org/docs/create-a-new-react-app.html/) to scaffold a new application.
```
Since `create-react-app` is no longer the recommended way to create a new app and it's not even in the official docs linked here, could we consider using another approach in this tutorial?
The recommended way is to use a framework, but they also mention Vite which I think is the simplest way to be used in this tutorial.

ekline[bot] committed:
docs: remove Okteto Cloud references and update the Flask/ReactJS tutorial

- Replace 'okteto login' with 'okteto context use'
- Replace deprecated 'okteto stack deploy' with 'okteto deploy'
- Point outdated okteto.com/docs links to current internal pages
- Replace cloud.okteto.com with okteto.example.com in CLI output

ekline[bot] <202747777+ekline[bot]@users.noreply.github.com>
